lib/stream_reader/librato_reporter.rb

```diff
@@ -17,6 +18,13 @@ def run!
     end
 
     private
+    
+    def config_present?
+      !ENV['LIBRATO_USER'].nil? &&
```

Comment:
Good call - I think the `!` gets called last but it doesn't hurt to be explicit. Fixed in https://github.com/ello/kinesis-stream-reader/pull/19/commits/8f1f4a8be9b8361c81dbd46ba3746fa099c0fe59